docker/alpine-latest/Dockerfile

```diff
@@ -15,6 +15,10 @@ ENV CI=$CI
 ENV DEBUG=$DEBUG
 ENV PRISMA_TELEMETRY_INFORMATION=$PRISMA_TELEMETRY_INFORMATION
 
+# Only OpenSSL 3 is expected to be on this system
```

Comment:
the file name says "latest" but in fact `node:lts-alpine` is used, so latest could be different than `lts-alpine` in theory I think

Should we use `current-alpine` instead here? https://hub.docker.com/_/node/

Reply:
Here `latest` refers to Alpine version and `lts` refers to Node.js version. I think it doesn't matter too much whether to use `current` or `lts` Node.js image here, but `current` might get newer Alpine earlier I guess. Let's use it instead.